packages/react-native-bpk-component-calendar/src/__snapshots__/BpkCalendar-test.ios.js.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`iOS BpkCalendar should render correctly 1`] = `
+<View
+  style={
+    Array [
+      Object {
+        "flex": 1,
+      },
+    ]
+  }
+/>
+`;
+
+exports[`iOS BpkCalendar should render correctly with arbitrary props 1`] = `
+<View
+  style={
+    Array [
+      Object {
+        "flex": 1,
+      },
+    ]
+  }
+  testID="123"
+/>
+`;
+
+exports[`iOS BpkCalendar should render correctly with custom style 1`] = `
+<View
```

Comment:
This looks a bit empty. Or is this just how Jest renders native code? I guess it would make sense that it doesn't render anything past the React code, because what would it put there..

Reply:
Fixed